### ISSUE TITLE
{"name":"TypeError","message":"Response constructor: Invalid response status code 204"} is thrown when browser is making a OPTIONS preflight request to an Elysia server running on Node.js

### ISSUE THREAD

**Issue by dtinth** [elysiajs/elysia-cors#47]
Node.js’s `Response` constructor throws an error when response body is not null (e.g. empty string):

```js
new Response('', { status: 204 })
/*
Uncaught TypeError: Response constructor: Invalid response status code 204
    at webidl.errors.exception (node:internal/deps/undici/undici:1636:14)
    at initializeResponse (node:internal/deps/undici/undici:5754:31)
    at new Response (node:internal/deps/undici/undici:5548:9)
*/
```

This is what `@elysiajs/cors` does:

- https://github.com/elysiajs/elysia-cors/blob/7ce6d9a64ff5df81d9be5b38c512c38b61fdb179/src/index.ts#L265-L267
- https://github.com/elysiajs/elysia-cors/blob/7ce6d9a64ff5df81d9be5b38c512c38b61fdb179/src/index.ts#L281-L283

Solution is to set `body` to `null` instead of `''`.

## Dependency versions

```json
    "@elysiajs/cors": "1.0.0-beta.0",
    "elysia": "1.0.0-beta.1",
```

**Comment by SaltyAom** [comment]
c069ff9 set response to null as suggested, published on 1.0.0-beta.1

Let me know if this fix the problem.

**Comment by dtinth** [comment]
[Tests are passing now](https://github.com/dtinth/remix-spa-frontend-elysia-backend/pull/9), thank you!
